internal/controller/vintagemc_controller.go

```diff
@@ -80,7 +81,7 @@ func (r *VintageMCReconciler) Reconcile(ctx context.Context, req ctrl.Request) (
 	loggedCluster := vintagemc.Object{
 		Object: service,
 	}
-	result, err = r.LoggingReconciler.Reconcile(ctx, loggedCluster)
+	_, err = r.LoggingReconciler.Reconcile(ctx, loggedCluster)
```

Comment:
Why are we ignoring result here?

Reply:
Was not used anyway, so it was creating errors at build.

Reply:
Alright

Reply:
Then apart from my 2 suggestions, I'm fine

Reply:
this could return `ctrl.Result` to controller whether the resource should be re-queued or not. Currently we do not need that.